src/braket/circuits/noise.py

```diff
@@ -165,9 +170,25 @@ def __eq__(self, other):
             return self.name == other.name and self.probability == other.probability
         return False
 
+    def bind_values(self, **kwargs) -> SingleProbabilisticNoise:
```

Comment:
You shouldn't have to keep defining bind_values as NotImplemented if it's not implemented in the base class (Noise). If it IS implemented in the base class, but you're overriding it here to be NotImplemented that seems not right.

Reply:
Good point. I think this was originally implemented, but I'll remove the method altogether.

Reply:
Going over it again, this is actually correct; `Noise` does not subclass `Parametrizable`, so these methods are the most abstract ones available.

Reply:
Oh, I'm sorry. That makes sense.